### PR TITLE
Draft blog post reporting EC activities

### PR DESCRIPTION
**Pull request by rgaiacs**
@carpentries/executive-council-2018 Anything else worth mention?

@mkuzak Could you review it? Thanks.

### CODE REVIEW

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+teaser: "Executive Council 1st 2018 Tertile Report: By laws"
+title: "Executive Council 1st 2018 Tertile Report"
+date: 2018-05-15
+time: "09:00:00"
+category: [ "Executive Council"]
+---
+
+May 2018 is almost over
+and The Carpentries Executive Council
+wants to provide a briefly report of their activities
```
Comment (k8hertweck) [inline _posts/2018/05/2018-05-15-ec-report.md]: briefly -> brief

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+activities were already shared
+but we still have a few minor details
+in operations that are orthogonal.
+The Executive Council is advising
+staffs in the best way to align operations
+of both organisations
+and facilitate the on boarding process
+of future member organisations.
+
+## By Laws
```
Comment (k8hertweck) [inline _posts/2018/05/2018-05-15-ec-report.md]: By laws -> Bylaws (here and in subsequent paragraph)

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+As a fiscally sponsored project of [Community Initiatives](http://communityin.org/),
+The Carpentries isn't required to have By Laws
+but the Executive Council believes that have this kind of document is important
+so most of the time in the previous four months were devoted to this activity.
+The new By Laws is inspired by Software Carpentry's one
+and other similar organisations.
+The Executive Council hopes to open a Request for Comments (RFC)
+around July after the in-person meeting.
```
Comment (k8hertweck) [inline _posts/2018/05/2018-05-15-ec-report.md]: Clarify here that the in-person meeting is for the Executive Council? Or put mention of this at the top of the post, to indicate a more general timeline?

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+authors: ["Raniere Silva", "Mateusz Kuzak"]
+teaser: "Executive Council 1st 2018 Tertile Report: By laws"
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: Bylaws is one word, I think.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Software Carpentry and Data Carpentry Operations
+
+At the time of the merge,
+most of Software Carpentry and Data Carpentry
+activities were already shared
+but we still have a few minor details
+in operations that are orthogonal.
+The Executive Council is advising
+staffs in the best way to align operations
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: staffs -> staff?

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+of both organisations
+and facilitate the on boarding process
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: onboarding

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+of future member organisations.
+
+## By Laws
+
+As a fiscally sponsored project of [Community Initiatives](http://communityin.org/),
+The Carpentries isn't required to have By Laws
+but the Executive Council believes that have this kind of document is important
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: have -> having

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+so most of the time in the previous four months were devoted to this activity.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: devoted to drafting the Bylaws for the new organisation

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+The new By Laws is inspired by Software Carpentry's one
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: is -> are

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+and other similar organisations.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: and those of other similar

**Comment by lexnederbragt** [comment]
Great to have this out soon!

* we should perhaps mention the work towards a new Mission & Vision before the Bylaws work is discussed
* Should we add that we have discussed The Carpentries financial situation and that we found it to be very healthy for the rest of 2018?
* Add link to http://static.carpentries.org/governance/ ?
* mention that most of the EC will be at CarpentryCon?

**Comment by kcranston** [comment]
Thanks for doing this, @rgaiacs ! @k8hertweck and @lexnederbragt have already mentioned everything that I saw, particularly including the other activities listed in Lex's comment.

**Comment by rgaiacs** [comment]
> we should perhaps mention the work towards a new Mission & Vision before the Bylaws work is discussed

I don't remember much about it from the top of my head. Could anyone write a paragraph?

> Should we add that we have discussed The Carpentries financial situation and that we found it to be very healthy for the rest of 2018?

Done.

> Add link to http://static.carpentries.org/governance/ ?

Done.

> mention that most of the EC will be at CarpentryCon?

Done.

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 2 above)
Comment (mkuzak) [reply _posts/2018/05/2018-05-15-ec-report.md]: lesson organisations?

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 7 above)
Comment (mkuzak) [reply _posts/2018/05/2018-05-15-ec-report.md]: By Laws -> bylaws
that **having** this kind of document

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 9 above)
Comment (mkuzak) [reply _posts/2018/05/2018-05-15-ec-report.md]: By Laws -> bylaws

**Comment by mkuzak** [comment]
thanks to all of you for this

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+time: "09:00:00"
+category: [ "Executive Council"]
+---
+
+May 2018 is almost over
+and The Carpentries Executive Council
+wants to provide a brief report of their activities
+during the first four months of 2018.
+
+Executive Council members are listed at [Governance page](http://static.carpentries.org/governance/).
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: Maybe add "where you can also find links to the minutes of the Executive Council monthly meetings"

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+and The Carpentries should be very healthy for the rest of 2018
+based on the projection.
+
+## Software Carpentry and Data Carpentry Operations
+
+At the time of the merge,
+most of Software Carpentry and Data Carpentry
+activities were already shared
+but we still have a few minor details
+in operations that are orthogonal.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: Could we simplify the language and say 'that are still different'

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+The Executive Council is advising
+staff in the best way to align operations
+of both organisations
+and facilitate the onboarding process
+of future member organisations.
+
+## Mission & Vision
+
+TODO
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: What about

"A major taks will be to determine the Mission and Vision statements of the new organisation. We are preparing a process where the community will be asked for suggestions. Based on this input, the Executive Council will prepare a draft text and collect feedback on it"

Could @tracykteal and @elliewix review this text as I am not sure it reflects the current plan.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+## Bylaws
+
+As a fiscally sponsored project of [Community Initiatives](http://communityin.org/),
+The Carpentries isn't required to have Bylaws
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: Bylaws (a set of operating rules for the new organisation)

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+but the Executive Council believes that having this kind of document is important
+so most of the time in the previous four months were devoted to drafting the Bylaws for the new organisation.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: were --> was

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+The new Bylaws are inspired by Software Carpentry's one
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: by the one from Software Carpentry and those of other similar non-profit organisations.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+and those of other similar.
+The Executive Council hopes to open a Request for Comments (RFC)
+around July after the Executive Council meet in-person on middle of May.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: we meet in the middle of June :-)

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+
+## Talk with us
+
+You can contact the Executive Council by [email](mailto:executive-council@carpentries.org).
+If you are attending CarpentryCon in two weeks time
+some of Executive Council members will be there
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: some --> most

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+and they will love to hear from you.
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: will --> would

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+date: 2018-05-15
+time: "09:00:00"
+category: [ "Executive Council"]
+---
+
+May 2018 is almost over
+and The Carpentries Executive Council
+wants to provide a brief report of their activities
+during the first four months of 2018.
+
```
Comment (lexnederbragt) [inline _posts/2018/05/2018-05-15-ec-report.md]: May I suggest adding something like "We are slowly getting to know each other, our role as Executive Council and what it means that we now are a single organisation"

**Review by lexnederbragt — COMMENTED** [review]
Thanks! I added yet another set of suggestions

**Comment by weaverbel** [comment]
Can someone please label this with `ready to merge` when you are all happy with it?" Then I can merge it. Thanks !

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 14 above)
Comment (rgaiacs) [reply _posts/2018/05/2018-05-15-ec-report.md]: Can you clarify this one?

**Comment by rgaiacs** [comment]
> Can someone please label this with ready to merge when you are all happy with it?

@tracykteal will do or merge it. We don't have writing permission so we can't add labels.

**Comment by lexnederbragt** [comment]
I'm more than happy for this to get merged :-)

**Comment by tracykteal** [comment]
Adding a few edits now. Please wait to merge.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 21]
```diff
@@ -0,0 +1,63 @@
...
+and what it means that we now are a single organisation.
+
+Executive Council members are listed at [Governance page](http://static.carpentries.org/governance/)
+where you can also find links to the minutes of the Executive Council monthly meetings.
+
+## Finances
+
+The first thing the Executive Councl had to look were the finances
+and The Carpentries should be very healthy for the rest of 2018
+based on the projection.
```
Comment (fmichonneau) [inline _posts/2018/05/2018-05-15-ec-report.md]: projections?

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 22]
```diff
@@ -0,0 +1,63 @@
...
+## Bylaws
+
+As a fiscally sponsored project of [Community Initiatives](http://communityin.org/),
+The Carpentries isn't required to have Bylaws
+but the Executive Council believes that having this kind of document is important
+so most of the time in the previous four months was devoted to drafting the Bylaws for the new organisation.
+The new Bylaws are inspired by the one from Software Carpentry
+and those of other similar non-profit organisations.
+The Executive Council hopes to open a Request for Comments (RFC)
+around July after the Executive Council meet in-person on middle of June.
```
Comment (fmichonneau) [inline _posts/2018/05/2018-05-15-ec-report.md]: on -> in the

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 23]
```diff
@@ -0,0 +1,63 @@
...
+time: "09:00:00"
+category: [ "Executive Council"]
+---
+
+May 2018 is almost over
+and The Carpentries Executive Council
+wants to provide a brief report of their activities
+during the first four months of 2018.
+We are slowly getting to know each other,
+our role as Executive Council
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: Executive Council -> the Executive Council

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 24]
```diff
@@ -0,0 +1,63 @@
...
+and what it means that we now are a single organisation.
+
+Executive Council members are listed at [Governance page](http://static.carpentries.org/governance/)
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: at -> on the

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 21 above)
Comment (tracykteal) [reply _posts/2018/05/2018-05-15-ec-report.md]: Executive Councl ... -> Executive Council reviewed were the finances, and The Carpentries are fiscally healthy through 2018 based on the projections.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 25]
```diff
@@ -0,0 +1,63 @@
...
+The first thing the Executive Councl had to look were the finances
+and The Carpentries should be very healthy for the rest of 2018
+based on the projection.
+
+## Software Carpentry and Data Carpentry Operations
+
+At the time of the merge,
+most of Software Carpentry and Data Carpentry
+activities were already shared
+but we still have a few minor details
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: shared -> shared,

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 26]
```diff
@@ -0,0 +1,63 @@
...
+in operations that are still different.
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: are still different -> are different

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 27]
```diff
@@ -0,0 +1,63 @@
...
+The Executive Council is advising
+staff in the best way to align operations
+of both organisations
+and facilitate the onboarding process
+of future member organisations.
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: future member organisations -> future lesson organisations.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 28]
```diff
@@ -0,0 +1,63 @@
...
+
+## Mission & Vision
+
+A major taks will be to determine the Mission and Vision statements of the new organisation.
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: A major taks ... -> A major task

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 29]
```diff
@@ -0,0 +1,63 @@
...
+We are preparing a process where the community will be asked for suggestions.
+Based on this input, the Executive Council will prepare a draft text and collect feedback on it.
+
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: Could expand on this a bit, to something like

"
We are preparing a process where the community will be asked for ideas and information. This input is key to developing the mission and vision of the organisation, and the Executive Council will use this information in developing draft text and plans for collecting and incorporating feedback.
"

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 14 above)
Comment (tracykteal) [reply _posts/2018/05/2018-05-15-ec-report.md]: I think the intent here is that not everyone knows what bylaws are, so we could define here, by including @lexnederbragt suggested text.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 30]
```diff
@@ -0,0 +1,63 @@
...
+## Bylaws
+
+As a fiscally sponsored project of [Community Initiatives](http://communityin.org/),
+The Carpentries isn't required to have Bylaws
+but the Executive Council believes that having this kind of document is important
+so most of the time in the previous four months was devoted to drafting the Bylaws for the new organisation.
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: is important so ... -> is important, so much of the time in the previous four months has been devoted to drafting ...

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 31]
```diff
@@ -0,0 +1,63 @@
...
+The new Bylaws are inspired by the one from Software Carpentry
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: one -> ones

File: _posts/2018/05/2018-05-15-ec-report.md (hunk 22 above)
Comment (tracykteal) [reply _posts/2018/05/2018-05-15-ec-report.md]: Executive Council meet ... -> Executive Council in-person meeting in the middle of June.

File: _posts/2018/05/2018-05-15-ec-report.md [hunk 32]
```diff
@@ -0,0 +1,63 @@
...
+and those of other similar non-profit organisations.
+The Executive Council hopes to open a Request for Comments (RFC)
+around July after the Executive Council meet in-person on middle of June.
+
+## Talk with us
+
+You can contact the Executive Council by [email](mailto:executive-council@carpentries.org).
+If you are attending CarpentryCon in two weeks time
+most of Executive Council members will be there
+and they would love to hear from you.
```
Comment (tracykteal) [inline _posts/2018/05/2018-05-15-ec-report.md]: and they would love -> and they would love to hear from and talk with you!

**Comment by tracykteal** [comment]
I added some suggestions for clarity and sentence structure. I closed a PR to @rgaiacs repo, because it brought in other changes, but the updates suggested are in this file https://github.com/tracykteal/carpentries.org/blob/tkt-ec/_posts/2018/05/2018-05-15-ec-report.md

**Comment by tracykteal** [comment]
@rgaiacs I can merge when you let me know here in the conversation to merge.

**Comment by rgaiacs** [comment]
> I added some suggestions for clarity and sentence structure.

Thanks.

> I can merge when you let me know here in the conversation to merge.

@tracykteal This is good to merge. Thanks.

**Comment by tracykteal** [comment]
Thanks @rgaiacs!

**Pull request merged**